pkgs/by-name/dm/dmlive/package.nix

```diff
@@ -10,21 +10,19 @@
   nodejs,
 }:
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage {
   pname = "dmlive";
-  version = "5.5.4";
+  version = "5.5.4-unstable-2025-01-25";
```

Comment:
`version` is `5.5.7`: https://github.com/THMonster/Revda/commit/e1c236f6f940443419b6202735b6f8a0c9cdbe8b

Reply:
`version` is exactly `5.5.7`, but it is not a big deal.  Feel free to merge it.

Reply:
The dmlive repo does not contain tags, so not really.